code/modules/food_and_drinks/food/customizables.dm

```diff
@@ -321,71 +328,98 @@
 /obj/item/reagent_containers/food/snacks/customizable/burger
 	name = "burger bun"
 	desc = "A bun for a burger. Delicious."
-	icon_state = "burger"
+	icon_state = "burgercustom"
 	baseicon = "burgercustom"
 	basename = "burger"
-	toptype = new /obj/item/reagent_containers/food/snacks/bun()
 	tastes = list("bun" = 4)
 
+
 /obj/item/reagent_containers/food/snacks/customizable/attackby(obj/item/I, mob/user, params)
-	if(contents.len > sandwich_limit)
+	if(!istype(I, /obj/item/reagent_containers/food/snacks))
+		to_chat(user, "<span class='warning'>[I] isn't exactly something that you would want to eat.</span>")
+		return
+
+	add_ingredient(I)
+
+/**
+ * Tries to add one ingredient and it's ingredients, if any and applicable, to this snack
+ * @param snack The ingredient that will be added
```

Comment:
`@param` isnt autodoc syntax
---
```
/**
  * Short description of the proc
  *
  * Longer detailed paragraph about the proc
  * including any relevant detail
  * Arguments:
  * * arg1 - Relevance of this argument
  * * arg2 - Relevance of this argument
  */

```

Reply:
Ah, I forgot which one it was again and found this one when searching for it. Will change